ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/txselection/BlockTransactionSelector.java

```diff
@@ -275,10 +315,14 @@ public TransactionSelectionResults evaluateTransactions(final List<Transaction>
    * @return The result of the transaction evaluation process.
    * @throws CancellationException if the transaction selection process is cancelled.
    */
-  private TransactionSelectionResult evaluateTransaction(
-      final PendingTransaction pendingTransaction) {
+  @Override
+  public TransactionSelectionResult evaluatePendingTransaction(
+      final org.hyperledger.besu.datatypes.PendingTransaction pendingTransaction) {
```

Comment:
Why do we need to import PendingTransaction with the whole package here ? is there another PendingTransaction ?

Reply:
yes, for example when we create the actual pending txs in the `evaluateTransactions` method, while this method is from the interface and uses the `PendingTransaction` interface